### PR TITLE
Add althea-dash to platform configs

### PR DESCRIPTION
**Pull request by drozdziak1**
**Note:** **Do not merge this!**
I intentionally put references to my forks here and in https://github.com/althea-mesh/althea-packages/pull/8 so that testing is a little bit easier. I'll switch everything back once review comes through

### CODE REVIEW

**Comment by drozdziak1** [comment]
FYI the fail above is virtualbox

File: roles/prepare-config/defaults/main.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
 source_dir: build/
 feeds_list:
-  - "src-git althea https://github.com/althea-mesh/althea-packages.git"
+  - "src-git althea https://github.com/drozdziak1/althea-packages.git;pre-0.1.0"
```
Comment (kingoflolz) [inline roles/prepare-config/defaults/main.yml]: shouldn't think still point to althea-mesh?

**Comment by jkilpatr** [comment]
I thought I removed the virtualbox profile?

File: config/ar750 [hunk 2]
```diff
@@ -24,6 +24,8 @@ CONFIG_OPENSSL_WITH_PSK=y
 CONFIG_OPENSSL_WITH_SRP=y
 CONFIG_PACKAGE_adblock=y
 CONFIG_PACKAGE_althea-babeld=y
+CONFIG_PACKAGE_althea-dash=y
+CONFIG_PACKAGE_althea-rust-binaries=y
```
Comment (jkilpatr) [inline config/ar750]: good catch

**Comment by jkilpatr** [comment]
https://github.com/althea-mesh/althea-firmware/commit/29a1ee150639129a5d7ba5831ec4ab1d5aa47de1

looks like I did but forgot to remove it from travis, derp

File: config/tplinkc7v2 [hunk 3]
```diff
@@ -7,6 +7,7 @@ CONFIG_OPENSSL_WITH_NPN=y
 CONFIG_OPENSSL_WITH_PSK=y
 CONFIG_OPENSSL_WITH_SRP=y
 CONFIG_PACKAGE_althea-babeld=y
+CONFIG_PACKAGE_althea-dash=y
```
Comment (drozdziak1) [inline config/tplinkc7v2]: FYI I remembered about the TP-LINK

**Pull request merged**